### PR TITLE
Kpm commands require valid package/command

### PR DESCRIPTION
**Pull request by victorhurdugaci**
Fixes https://github.com/aspnet/XRE/issues/1222 and https://github.com/aspnet/XRE/issues/1223

cc @davidfowl

### CODE REVIEW

File: src/Microsoft.Framework.PackageManager/Restore/RestoreCommand.cs [hunk 1]
```diff
@@ -288,14 +288,17 @@ public RestoreCommand(IApplicationEnvironment env)
...
+            if (appLib == null)
             {
-                DefaultPackagePathResolver resolver = new DefaultPackagePathResolver(packagesDirectory);
-                
-                // The call to GetFullPath is important because GetInstallPath returns a path with a "\.\" in it
-                AppInstallPath = Path.GetFullPath(resolver.GetInstallPath(appLib.Name, appLib.Version));
+                return false;
             }
 
+            DefaultPackagePathResolver resolver = new DefaultPackagePathResolver(packagesDirectory);
```
Comment (Praburaj) [inline src/Microsoft.Framework.PackageManager/Restore/RestoreCommand.cs]: `var`

File: src/Microsoft.Framework.PackageManager/Restore/RestoreCommand.cs [hunk 2]
```diff
@@ -320,8 +323,8 @@ public RestoreCommand(IApplicationEnvironment env)
             var lockFile = await ReadLockFile(projectLockFilePath);
 
             var useLockFile = false;
-            if (Lock == false && 
-                Unlock == false && 
+            if (Lock == false &&
```
Comment (Eilon) [inline src/Microsoft.Framework.PackageManager/Restore/RestoreCommand.cs]: There's a simpler way to evaluate `bool`s :smile: (The code was already there, I know.)

Comment (victorhurdugaci) [inline src/Microsoft.Framework.PackageManager/Restore/RestoreCommand.cs]: `If (!!(Lock==false) && !!(!Unlock == true))` ? :trollface:

Comment (Eilon) [inline src/Microsoft.Framework.PackageManager/Restore/RestoreCommand.cs]: Definitely :trollface:

**Comment by troydai** [comment]
@victorhurdugaci could you please rebase and force push your changes again? the incident this morning disrupted all the PRs.

**Comment by victorhurdugaci** [comment]
@troydai Rebased

**Comment by troydai** [comment]
:+1:

**Comment by victorhurdugaci** [comment]
@troy is :+1: a soft :shipit: ?

**Comment by troy** [comment]
Wrong @troy :)

**Comment by victorhurdugaci** [comment]
Whoops, sorry :) Adding the correct Troy @troydai

**Comment by troydai** [comment]
:shipit:

**Pull request merged**